x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/pinned/index.tsx

```diff
@@ -236,38 +263,54 @@ export const PinnedTabContentComponent: React.FC<Props> = ({
     onEventClosed({ tabType: TimelineTabs.pinned, id: timelineId });
   }, [timelineId, onEventClosed]);
 
-  if (unifiedComponentsInTimelineEnabled) {
+  const NotesFlyoutMemo = useMemo(() => {
```

Comment:
Does this work if you just pass the component after the first opening React fragment, so it exists for the unified flag enabled and disabled without the `useMemo`?

Reply:
I have enabled `Notesflyout` as the default behaviour. With or without the `unifiedComponentsInTimelineEnabled` flag.  As you can see [here](https://github.com/elastic/kibana/pull/186948/files/78662badd8dfa74812a7231a442ed366774e9f34#diff-b05096c9bd235751d50e629aeba48e8f1254b38e41f7469b796eab4367f6bcabL351-L365), the notes will never appear in the row. I have removed it from old timeline table as well. 

I was wanted to keep this consistent. Let me know if you disagree.

Reply:
Oh yea, having it as the default behavior is fine, I just didn't think it needed to be inside a `useMemo` but could rather be in-line outside the ternary conditional. It's more of a thought/preference than something that needs to be changed. 👍🏾